lib/cocoapods/installer/analyzer/sandbox_analyzer.rb

```diff
@@ -223,6 +230,28 @@ def sandbox_checksum(pod)
           sandbox_manifest.checksum(pod)
         end
 
+        # @return [Dependency] The dependency with the given name stored in the sandbox.
+        #
+        # @param  [String] pod
+        #         the name of the Pod.
+        #
+        def sandbox_dependency(pod)
+          sandbox_manifest.dependencies.each { |d| return d if d.name == pod }
```

Comment:
reeplace with:

```
sandbox_manifest.dependencies.find { |d| d.name == pod }
```

and remove the `return nil` below.

Reply:
done. thanks for your review